core/scss/colors/utilities.scss

```diff
@@ -0,0 +1,391 @@
+//------------------------------------------------------
+// Do not edit this file directly. These utility classes
+// are generated from the Momentum Design color tokens.
+// @collab-ui/core/tokens/colors.json
+//------------------------------------------------------
+
+.cui-color--theme-10 { color: #E6F9FC !important; }
+.cui-color--theme-20 { color: #B8F2FF !important; }
+.cui-color--theme-30 { color: #73E3FF !important; }
+.cui-color--theme-40 { color: #1DC4F2 !important; }
+.cui-color--theme-50 { color: #00A0D1 !important; }
+.cui-color--theme-60 { color: #007EA8 !important; }
+.cui-color--theme-70 { color: #0A5E7D !important; }
+.cui-color--theme-80 { color: #124052 !important; }
+.cui-color--theme-90 { color: #122933 !important; }
+.cui-color--theme-100 { color: #101C21 !important; }
+
+.cui-color--blue-10 { color: #E6F9FC !important; }
+.cui-color--blue-20 { color: #B8F2FF !important; }
+.cui-color--blue-30 { color: #73E3FF !important; }
+.cui-color--blue-40 { color: #1DC4F2 !important; }
+.cui-color--blue-50 { color: #00A0D1 !important; }
+.cui-color--blue-60 { color: #007EA8 !important; }
+.cui-color--blue-70 { color: #0A5E7D !important; }
+.cui-color--blue-80 { color: #124052 !important; }
+.cui-color--blue-90 { color: #122933 !important; }
+.cui-color--blue-100 { color: #101C21 !important; }
```

Comment:
Do we want to keep cui-color--blue with theme being the same.

Reply:
Yes